.travis.yml

```
@@ -39,6 +40,9 @@ jobs:
     ###########################################
 
     # Disable Async Hooks
```

Test, please ignore me:

```suggestion
    # Disable Async-Hooks
```

---

Made this review to help github staff fix the a bug in the suggested change feature - just ignore